README.md

```diff
@@ -72,10 +78,32 @@ This repo contains the following packages
 
 -   Private package providing test utilities for `@sa11y` packages
 
-#### [Integration Tests](packages/test-integration/README.md)
+#### [Integration Tests](./packages/test-integration/README.md)
 
 -   Private package providing integration tests for `@sa11y` packages
 
-#### [Common](packages/common/README.md)
+#### [Common](./packages/common/README.md)
 
 -   Common utilities, constants, error messages for `@sa11y` packages
+
+### Dependency graph
+
+![Dependency graph of sa11y packages](https://github.com/salesforce/sa11y/blob/media/architecture/sa11y_dependency_graph.svg?raw=true)
+
+---
+
+### Epilogue
```

Comment:
@cordeliadillon Can you please review if this looks good? 4

Does an `Epilogue` make sense? Or should the section title be `A wise person once said…`?